src/tips/topics.json

```diff
@@ -22,5 +22,9 @@
     {
         "id": "console-err-ff",
         "title": "How to check for errors in Firefox console"
+    },
+    {
+        "id": "configue-togg-auto",
+        "title": "Configue site toggling and Automation"
```

Comment:
"Configu**r**e ... **a**utomation" or *"Configure Automation and Site Toggling"* or *"How to configure Automation and Site Toggling"*.